pkg/config/constants/constants.go

```diff
@@ -108,5 +108,18 @@ const (
 	// DefaultConfigServiceAccountName is the default service account to use for external Istiod cluster access.
 	DefaultConfigServiceAccountName = "istiod-service-account"
 
+	// KubeNamespaceSystem is the system namespace where we place kubernetes system components.
+	KubeNamespaceSystem string = "kube-system"
+
+	// KubeNamespacePublic is the namespace where we place kubernetes public info (ConfigMaps).
+	KubeNamespacePublic string = "kube-public"
```

Comment:
```suggestion
	KubePublicNamespace string = "kube-public"
```

and like wise for other namespaces.